draft-httpbis-doh-preference-hints.md

```diff
@@ -159,6 +159,21 @@ of the web server. Web servers MAY send multiple DoH-Preference headers. Web
 servers MUST NOT send the DoH-Preference header in HTTP responses conveyed over
 a non-secure transport.
 
+## Considerations For Choosing a Preferred DoH Server
+
+The choice of DoH server can affect overall performance and responsiveness as
+perceived by the client. Some example considerations in choosing a preferred
+DoH server are:
+
+* A DoH host specified as a host name rather than an IP address will require
```

Comment:
Since a DoH server needs a TLS certificate, doesn't it need a host name? I forgot what the current status of IP address TLS certs are...

Reply:
IP address TLS certs are fine and permitted. https://1.1.1.1, for example

Reply:
Thanks @grittygrease